src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise-update.component.html

```diff
@@ -1,13 +1,100 @@
 <div class="row justify-content-center">
     <div class="col-8">
         <form name="editForm" role="form" novalidate (ngSubmit)="save()" #editForm="ngForm">
-            <h2 id="jhi-file-upload-exercise-heading" jhiTranslate="artemisApp.fileUploadExercise.home.createOrEditLabel">Create or edit a File Upload Exercise</h2>
+            <h2 *ngIf="fileUploadExercise.id" id="jhi-file-upload-exercise-heading-edit">{{ 'artemisApp.fileUploadExercise.home.editLabel' | translate }}</h2>
```

Comment:
You can also use `jhiTranslate="artemisApp..." on the tag.

Reply:
Do you think this is a better approach? In different places we use different approaches. I personally, prefer the one I use, as it is shorter.

Reply:
After some clarification I decided to use jhiTranslate here